docs/design/storage-node-windows-installer.md

```diff
@@ -13,73 +13,66 @@ Similarly, Docker ends up using more resources than running natively on Windows.
-
-This means that Windows handles starting and restarting the binary in the background.
+The high-level idea is to create an installer using the [WiX toolset](https://wixtoolset.org) and make Storage Node a Windows service.
+
+The installer shall:
+* Display a GUI for collecting user configuration about:
+  * Wallet address
+  * Email
+  * External address/port
+  * Advertised bandwidth 
+  * Advertised Storge
+  * Identity directory
+  * Installation directory
+  * Storage directory
+* Install the storage node binary in the installation directory.
+* Generate `config.yaml` file with the user configuration.
+* Register the storage node binary as a Windows Service: https://wixtoolset.org/documentation/manual/v3/xsd/util/serviceconfig.html
+* Create a firewall exception for port 28967: https://wixtoolset.org/documentation/manual/v3/xsd/firewall/firewallexception.html
+* Create shortcut for opening the Dashboard
+* Add installation directory to user PATH (optional)
+* Open the dashboard when the installation is complete (optional)
+* Install the auto-updater binary and run it as a Windows service (if auto-updater is implemented at this point)
+
+We must ensure that both the storage node binary and the MSI installer are signed with Storj code-sign certificate to avoid warning popups to users.
+
+## Rationale
+
+The initial idea was to use [go-msi](https://github.com/mh-cbon/go-msi) for creating the Windows installer. It utilizes the WiX toolset to build the actual installer.
+
+There are a number of reasons to use the WiX toolset directly instead of go-msi:
+* More flexibility as we can access all WiX toolset features directly instead of through the go-msi wrapper
+* One less dependency
+* The go-msi project is stall for 2 years. None of the issues and question has been answered for the last year.
+* The go-msi project lacks support for Windows Services and Firewall Rules, although the WiX toolset natively supports them.
 
 ## Implementation
 
-1) Modify the storage node startup to implement ServiceMain so that the binary is considered a Windows Service.
-2) Create script for registering binary as a Windows Background Service. (sc.exc command)
-3) Create wix.json file for building an MSI.
-4) Update the build process to include the creation of the MSI. 
-5) Ensure the windows installer is working properly
-  * Install Storage Node using msi.
+1. Implement MSI installer using the WiX toolset.
+1. Update the build process to build and sign the MSI installer. 
+1. Ensure the MSI installer works properly
+  * Install Storage Node using the MSI installer.
   * Ensure binaries run on Windows startup, and it runs as a background process when not logged in.
   * Ensure UAC(s) is not triggered after installation.
   * Ensure that storage node restarts automatically after a crash 
 
 ## Open Issues/ Comments (if applicable)
```

Comment:
I think that `(if applicable)` is meant to be removed from the template when the section is present in the design doc.